azure-media/src/main/java/com/microsoft/windowsazure/services/media/models/Operation.java

```diff
@@ -37,7 +37,7 @@ private Operation() {
     /**
      * Create an operation object that will get the state of the given asset.
      * 
-     * @param assetId
+     * @param operationId
```

Comment:
this should be AssetId still right? Not the Operation ID

Reply:
Fixed